### PR TITLE
Multiple tables

### PR DESCRIPTION
**Pull request by lengushuru**
- Added two new tables, owners and species

- Modified animals table

- Inserted data into the owners and species table

- the following are screen shots for queries on vet_clinic database
What animals belong to Melody Pond?
melody
List of all animals that are pokemon (their type is Pokemon).
pokemon

List all owners and their animals, remember to include those that don't own any animal.
list-of-owners
List all Digimon owned by Jennifer Orwell.
owned-jeniffer

List all animals owned by Dean Winchester that haven't tried to escape.
own-dean
Who owns the most animals?
own-most

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -9,4 +9,23 @@ CREATE TABLE animals (
...
+);
+
+CREATE TABLE species (
+    id BIGSERIAL primary key, 
+    name varchar(255)
+);
+
+ALTER TABLE animals DROP column species;
+ALTER TABLE animals ADD species_id INT;
+ALTER TABLE animals ADD FOREIGN key(species_id) REFERENCES species(id);
```
Comment (muneebulrehman) [inline schema.sql:27]: - Optional
You can do the same in one line. ⬇️
`ALTER TABLE animals ADD species_id INT REFERENCES species(id);`

File: data.sql [hunk 2]
```diff
@@ -7,4 +7,31 @@ VALUES
...
+
+update animals set species_id = case 
+  when name like '%mon' then (select id from species where name = 'Digimon')
+  else (select id from species where name = 'Pokemon')
+end;
+
+update animals set owner_id = case
+  when name = 'Agumon' then (select id from owners where full_name = 'Sam Smith')
+  when name = 'Pikachu' or name = 'Gabumon'  then (select id from owners where full_name = 'Jesnnifer Orwell')
+  when name = 'Plantmon' then (select id from owners where full_name = 'Bob')
```
Comment (muneebulrehman) [inline data.sql]: As per the project requirements, Bob owns two animals. *Plantmon* and *Devimon*. However, you have taken only one into account. Kindly fix this error. You will have to update the query results accordingly and update the screenshots where necessary. ⬇️
image

File: queries.sql [hunk 3]
```diff
@@ -54,4 +48,40 @@ GROUP BY species;
...
+WHERE owners.full_name = 'Melody Pond';
+
+SELECT animals.name FROM animals
+JOIN species ON animals.species_id = species.id
+WHERE species.name = 'Pokemon';
+
+
+SELECT owners.full_name, animals.name
+FROM owners
+LEFT JOIN animals ON owners.id = animals.owner_id;
```
Comment (muneebulrehman) [inline queries.sql:64]: The result of the corresponding screenshot to this query is not correct. Eg., *Gabumon* is missing from the list and so is *Pikachu* along with *Devimon*. Kindly fix this error.

File: queries.sql [hunk 4]
```diff
@@ -54,4 +48,40 @@ GROUP BY species;
...
+SELECT species.name, COUNT(animals.id) as count
+FROM species
+LEFT JOIN animals ON species.id = animals.species_id
+GROUP BY species.id;
+
+SELECT animals.name
+FROM animals
+JOIN species ON animals.species_id = species.id
+JOIN owners ON animals.owner_id = owners.id
+WHERE species.name = 'Digimon' AND owners.full_name = 'Jennifer Orwell';
```
Comment (muneebulrehman) [inline queries.sql:74]: The result of the corresponding screenshot to this query in your PR is not correct. **Jennifer Orwell** has one **Digimon**, *Gabumon*  and your results show none. Kindy fix this error.

File: queries.sql [hunk 5]
```diff
@@ -54,4 +48,40 @@ GROUP BY species;
...
+
+
+SELECT owners.full_name, animals.name
+FROM owners
+LEFT JOIN animals ON owners.id = animals.owner_id;
+
+SELECT species.name, COUNT(animals.id) as count
+FROM species
+LEFT JOIN animals ON species.id = animals.species_id
+GROUP BY species.id;
```
Comment (muneebulrehman) [inline queries.sql]: The result's screenshot for this query is missing. Kindly include it in your PR before submitting it for the re-review.

**Review by muneebulrehman — CHANGES_REQUESTED** [review]
Hi @lengushuru , 

Good job so far! I see you put a lot of effort into this project. However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- No linter errors
- Descriptive PR
- Great job reaching so far. 👍

## Required Changes ♻️
_Check the comments under the review._

- [ ] In your README file, please include the section **Key Features** from t[his provided template](https://github.com/microverseinc/readme-template). Remember the only optional section from this template file is *FAQ section**. All the rest of the main headers should be present in your repo.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 5 above)
Comment (lengushuru) [reply queries.sql]: Thanks for review, I will be working to fix requested changes 👍

**Review by mwanawabangona — APPROVED** [review]
Hi @lengushuru 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations!  


## Highlights
*   No linter errors :heavy_check_mark:
*   Repo looks professional :briefcase:
*   Solid pull request scroll
*   Good quality commits
*   Good job implementing the requested change!! :clap: :clap: :clap:

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**